src/Control/Effect/Choose.hs

```diff
@@ -11,7 +11,7 @@ module Control.Effect.Choose
 , Choosing(..)
 ) where
 
-import Control.Carrier
+import {-# SOURCE #-} Control.Carrier
```

Comment:
These directives indicate that it’s `Control.Carrier` that gets a `.hs-boot` file, and not, say, this module.